INSTALL.md

```diff
@@ -12,6 +12,16 @@ source.
     $ make
     $ sudo make install
 
+On RHEL, Fedora and CentOS machines, following commands will build and install uftrace from source. 
+
+    $ sudo yum install elfutils-devel	# for debug info       (optional)
+    $ sudo yum install pandoc		# for man pages        (optional)
+    $ sudo yum install python2-devel	# for python scripting (optional)
+    $ sudo yum install ncurses-devel	# for TUI              (optional)
+    $ sudo yum install pkgconf-pkg-config	# for configure        (optional)
```

Comment:
Does this work all for recent Fedora, RHEL and CentOS?  I'm wondering `dnf` is not working on RHEL yet.

Reply:
I just tested it (using dnf) on Fedora 28 (Twenty Eight) and it works fine. But not yet confirmed for other distros.